db/schema.rb

```diff
@@ -206,6 +206,7 @@
     t.integer  "position"
     t.datetime "created_at",          null: false
     t.datetime "updated_at",          null: false
+    t.text     "edm_is_shown_by",     null: false
```

Comment:
Given that we plan to introduce the feature to also be able to specify a `hasView` image from a record, should we instead name this column `url`?